docs/commandline/tradebot.md

```diff
@@ -6,11 +6,11 @@ You can set Gekko up as a tradebot, this will instruct Gekko to:
 - Run a strategy (in semi-realtime) over live market data.
 - Automatically create buy/sell orders based on signals coming from the strategy.
 
-*As with everything in Gekko, the tradebot will make decisions based on the by YOU selected/configured/created strategy. If you end up losing money you have no one to blame but yourself.*
+*As with everything in Gekko, the tradebot will make decisions based on the selected/configured/created strategy. If you end up losing money, you have no one to blame but yourself.*
```

Comment:
I think the `by YOU` is an emphasis that we shouldn't remove. If the sentence structure is currently wrong, can we fix it instead of removing this?

Reply:
IMO the emphasis on the `YOU` should stay. Maybe sth like `As with everything in Gekko, the tradebot will make decisions based on strategies, **YOU** select/configure/created.`?